SUMMARY.md

```diff
@@ -442,6 +453,7 @@
 
 - [Contributing to Hardware](./doc/contributing/hw/README.md)
   - [Comportability](./doc/contributing/hw/comportability/README.md)
+    - [Comportable IP (CIP) Testbench Architecture](./hw/dv/sv/cip_lib/README.md)
```

Comment:
Did we agree that we should only have one of these copies?

Reply:
Yes I will remove the second copy.